public-api/src/main/java/org/pmiops/workbench/cdr/dao/DbDomainDao.java

```diff
@@ -1,10 +1,13 @@
 package org.pmiops.workbench.cdr.dao;
 import org.pmiops.workbench.cdr.model.DbDomain;
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 
 import java.util.List;
 
 public interface DbDomainDao extends CrudRepository<DbDomain, Long> {
     // TODO -- maybe add order by
     List<DbDomain> findAll();
+
+    List<DbDomain> findByDbTypeAAndConceptIdNot(String db_type,long conceptId);
```

Comment:
Once conceptId is null everywhere it is zere , make this findByDbTypeAndConceptIdIsNotNull(db_type) ..
There is a typo in this one with the "AAnd" isn't there?